src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -192,6 +199,11 @@ function NameAndPicture({ navigation, route }: Props) {
     }
   }
 
+  const onPressGenerateUsername = () => {
+    //TODO: Obtain forbidden words from Firebase Remote Config
```

Comment:
I'm seeing that https://github.com/valora-inc/wallet/pull/2974 is approved, so this will be possible very soon

Reply:
Yeah just merged it.